winui/AutoComplete/Selection.md

```diff
@@ -98,6 +98,40 @@ autoComplete.SelectionMode = AutoCompleteSelectionMode.Multiple;
 
 ![Multiple selection in WinUI AutoComplete](Selection_images/winui-autocomplete-multiple-selection.gif)
 
+## Auto-Append UI
+
+The AutoComplete control provides auto-append support with the selection of text as well as text alone. The auto-append UI can be defined by using the `AppendType` property. There are two types of auto-append UI: `TextWithSelection` and `Text`. When the auto-append UI is set to `Text`, the appended text appears with a faded foreground, similar to Windows 11.
```

Comment:
The AutoComplete control provides auto-append support with the selection of text as well as text alone. The auto-append UI can be defined by using the `AppendType` property. There are two types of auto-append UI: the `TextWithSelection` and `Text.` When the auto-append UI is set to `Text,` the appended text appears with a faded foreground, similar to Windows 11.

Reply:
Updated